### PR TITLE
fix: upgrade template apps to v11

### PR DESCRIPTION
**Pull request by shaftoe**
- For each template:
  - fix deprecation warnings upgrading `package.json` + fixing units for latest Probot version usage
  - fix app signatures, ref: https://github.com/probot/probot/issues/1286#issuecomment-744094299
- tentative small changes to `README.md` meant to reduce frictions for new users

Note to myself: final welcome message for TypeScript users should remind about the required `build` step, I'll open a dedicated issue for it

-----
[View rendered README.md](https://github.com/shaftoe/create-probot-app/blob/upgrade-to-v11/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,19 +2,28 @@
...
-```
+## Installation
 
-If you're using Yarn:
+Make sure you've got [NodeJS installed](https://nodejs.org/en/download/) on your workstation, than open your terminal and type the following command:
 
-```sh
-yarn create probot-app my-first-app
-```
+- if you're using NPM:
```
Comment (gr2m) [inline README.md:13]:
```suggestion
- if you're using `npm` (bundled with `Node.js`)
```

File: README.md [hunk 2]
```diff
@@ -2,19 +2,28 @@
...
-In particular, this command line interface allows you to select from our pre-defined blue prints to choose a basic working example to start from.
+More specifically, this command line interface allows you to select from our pre-defined blue prints to choose a basic working example to start from.
 
-```sh
-npx create-probot-app my-first-app
-```
+## Installation
 
-If you're using Yarn:
+Make sure you've got [NodeJS installed](https://nodejs.org/en/download/) on your workstation, than open your terminal and type the following command:
```
Comment (gr2m) [inline README.md:11]:
```suggestion
Make sure you've got [Node.js installed](https://nodejs.org/en/download/) on your workstation, than open your terminal and type the following command:
```

**Review by gr2m — COMMENTED** [review]
this looks great, just two minor notes on the README update

**Comment by shaftoe** [comment]
This also rises the question of how to ensure these files are always up to date with the latest APIs.

If I get it right, every time there's a bump in the major Probot version there's a need for this kind of intervention, at least to update the `package.json` files. Is `dependabot` already able to help with it? should we add an automated task to open an issue any time a major version is out?

**Comment by gr2m** [comment]
> Is `dependabot` already able to help with it? should we add an automated task to open an issue any time a major version is out?

> Is `dependabot` already able to help with it? should we add an automated task to open an issue any time a major version is out?

Hmm I think it should, can you try adding one entry per template directory? 
https://github.com/probot/create-probot-app/blob/bff761f0a1c7e11a6a3c9131a84af7d19f48d599/.github/dependabot.yml#L3-L6

Not ideal but I think that's how it works.

Alternatively we could create a GitHub workflow that acts on push events to branches starting with `dependabot/npm_and_yarn/probot-` and update all `templates/*/package.json` files with the same probot version, but that would only update `probot` in the template folders, while it should update all dependencies.

**Comment by gr2m** [comment]
it's outside of the scope of this repository, so I'm going to merge this.

**Review by gr2m — APPROVED** [review]
Thanks Alex! 💐

**Pull request merged**